extensions/amp-next-page/1.0/service.js

```diff
@@ -840,52 +866,160 @@ export class NextPageService {
+    return this.getSeparatorContent_(separator, page).then(rendered => {
+      return this.mutator_.mutateElement(separator, () => {
+        removeChildren(dev().assertElement(separator));
+        separator.appendChild(rendered);
+      });
+    });
+  }
+
+  /**
+   * Creates the internal separator content based on the
+   * given information about the next page
+   * @param {!Element} separator
+   * @param {!Page} page
+   * @return {!Promise}
+   */
+  getSeparatorContent_(separator, page) {
     const data = /** @type {!JsonObject} */ ({
       title: page.title,
       url: page.url,
       image: page.image,
     });
-    return this.templates_
-      .findAndRenderTemplate(separator, data)
-      .then(rendered => {
-        return this.mutator_.mutateElement(separator, () => {
-          removeChildren(dev().assertElement(separator));
-          separator.appendChild(rendered);
-        });
-      });
+
+    // In the case of a templated separator, render the internal template
+    if (this.templates_.hasTemplate(separator)) {
+      return this.templates_.findAndRenderTemplate(separator, data);
+    }
+
+    // Otherwise this is a default separator
+    devAssert(this.hasDefaultSeparator_);
+
+    const content = this.doc_.createElement('div');
+    content.classList.add('amp-next-page-separator-content');
+
+    const image = this.doc_.createElement('img');
+    image.classList.add('amp-next-page-separator-img');
+    image.src = data['image'];
+    content.appendChild(image);
+
+    const title = this.doc_.createElement('span');
+    title.classList.add('amp-next-page-separator-title');
+    title.textContent = `Next article: ${data['title']}`;
+    content.appendChild(title);
+
+    return Promise.resolve(content);
   }
 
   /**
    * @param {!Element} element the container of the amp-next-page extension
    * @return {!Element}
    * @private
    */
-  getMoreBoxElement_(element) {
-    const providedMoreBox = childElementByAttr(element, 'more-box');
-    // TODO(wassgha): Use templates (amp-mustache) to render the more box
-    if (providedMoreBox) {
-      removeElement(providedMoreBox);
+  getFooterElement_(element) {
+    const providedFooter = childElementByAttr(element, 'footer');
+    if (providedFooter) {
+      removeElement(providedFooter);
+      return providedFooter;
     }
-    return providedMoreBox || this.buildDefaultMoreBox_();
+    // If no footer is provided then we build a default one
+    this.hasDefaultFooter_ = true;
+    return this.buildDefaultFooter_();
   }
 
   /**
    * @return {!Element}
    * @private
    */
-  buildDefaultMoreBox_() {
-    // TODO(wassgha): Better default more box
-    const moreBox = this.win_.document.createElement('div');
-    moreBox.classList.add('amp-next-page-default-more-box');
-    return moreBox;
+  buildDefaultFooter_() {
+    const html = htmlFor(this.getHost_());
+    return html`
+      <div class="amp-next-page-footer" aria-label="Read more articles"></div>
```

Comment:
What happens with i18n for this aria-label?

Reply:
Blocked by https://github.com/ampproject/amphtml/issues/4146 I don't think there's a nice way to do it so far. Lightbox and carousel controls do the same.

Reply:
Let's expose this as an optional `data-read-more-label` then.